src/mcpbr/benchmarks/codecontests.py

```
@@ -0,0 +1,250 @@
+        temp_task = {
+            "instance_id": instance_id,
+            "repo": "codecontests/competitive",
+            "base_commit": "HEAD",
+        }
+        return await docker_manager.create_environment(temp_task)
+
+    async def evaluate(
+        self,
+        env: TaskEnvironment,
+        task: dict[str, Any],
+        solution: str,
```

Ruff (0.14.14)

[warning] Unused method argument: `solution`

(ARG002)